packages/fx-core/tests/core/middleware/testAssets/v3Migration/debug/V3.5.0-V4.0.6-command/expected/app.local.yml

```diff
@@ -1,55 +1,30 @@
-# yaml-language-server: $schema=https://aka.ms/teams-toolkit/1.0.0/yaml.schema.json
-# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
-# Visit https://aka.ms/teamsfx-actions for details on actions
-version: 1.0.0
+{{header}}
```

Comment:
the expected content seems not right

Reply:
Since there are duplicated common yml snippets in many unit cases, we use templates to replace them. 
template content like {{header}} is in the yml folder testAssets\v3Migration\ymlTemplates.